packages/iov-keycontrol/src/keyring.spec.ts

```diff
@@ -121,7 +124,7 @@ describe("Keyring", () => {
         "wallets": [
           {
             "implementationId": "ed25519-hd",
-            "data": "{\\"formatVersion\\":1,\\"id\\":\\"f49h7gh34\\",\\"secret\\":\\"side ripple bachelor banner word swear buzz try situate rent desk carry scorpion uphold undo account pumpkin throw\\",\\"curve\\":\\"ed25519 seed\\",\\"identities\\":[]}"
+            "data": "{\\"formatVersion\\":2,\\"id\\":\\"f49h7gh34\\",\\"secret\\":\\"side ripple bachelor banner word swear buzz try situate rent desk carry scorpion uphold undo account pumpkin throw\\",\\"curve\\":\\"ed25519 seed\\",\\"identities\\":[]}"
```

Comment:
Hehehe... time to see our migrations in use.
Let's see if there are tests from migrating version 1 to version 2....

Reply:
Unfortunatly no migration possible due to missing information